check_copyright.py

```diff
@@ -518,14 +544,18 @@ def build_parser() -> argparse.ArgumentParser:
     parser.add_argument('-d', '--debug', action='store_true',
                         help='print debug info')
     parser.add_argument('-du', '--dont-update-ignore-list', action='store_true')
+    parser.add_argument('-co', '--check-only', action='store_true', help='check without adding new headers')
+    parser.add_argument('-i', '--ignore', default='check_copyright_ignore', help='set path to the ignore list')
```

Comment:
The readme could explain these inputs in more detail. For example, what is an ignore list (format, meaning, ...).